### PR TITLE
feat(releases): Releases sort dropdown options syncs with display

### PR DESCRIPTION
**Pull request by taylangocmen**
This updates the release health page with the changes to the releases page dropdown that were in:

https://github.com/getsentry/sentry/pull/26804
https://github.com/getsentry/sentry/pull/26853

### CODE REVIEW

File: src/docs/product/releases/health/index.mdx [hunk 1]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
...
+### Active Sessions / Users
 
-Users who started the application at least once in the last 24 hours.
+Active Sessions are sessions that started in the application in the last 24 hours.
+
+Active Users are users who started the application at least once in the last 24 hours.
+
+To sort releases by active sessions, select Display: Sessions and to sort by active users select Display: Users.
+
+![Sort By: Active Sessions / Users dropdown](release_active.png)
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:37]: This screenshot is a confusing. Recommend you split them into two and place one after the other.

File: src/docs/product/releases/health/index.mdx [hunk 2]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
...
 
-### Active Users
+### Active Sessions / Users
 
-Users who started the application at least once in the last 24 hours.
+Active Sessions are sessions that started in the application in the last 24 hours.
+
+Active Users are users who started the application at least once in the last 24 hours.
+
+To sort releases by active sessions, select Display: Sessions and to sort by active users select Display: Users.
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:35]:
```suggestion
Depending on your your "Display" selection, the sort options for the page change. To sort releases by active sessions, select "Sessions" in the "Display" dropdown, and to sort by active users, select "Users".
```

File: src/docs/product/releases/health/index.mdx [hunk 3]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
 
 Sessions are submitted to Sentry so you can track the usage and adoption of your application. When a user of your application experiences a crash, error, or abnormal exit, the session will be flagged accordingly, and Sentry calculates derived metrics. The metrics include data such as the number of users that didn't experience a crash in the specified time range.
 
-### Active Users
+### Active Sessions / Users
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:29]:
```suggestion
### Active Sessions/Users
```

File: src/docs/product/releases/health/index.mdx [hunk 4]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
...
+Crash Free Sessions is the percentage of sessions in the specified time range not ended by a crash of the application.
 
-Percentage of distinct users who did not experience a crash during the specified time period.
+Crash Free Users is the percentage of distinct users who did not experience a crash during the specified time period.
 
-### Crash Free Sessions
+To sort releases by crash free sessions, select Display: Sessions and to sort by crash free users select Display: Users.
 
-Percentage of sessions in the specified time range not ended by a crash of the application.
+![Sort By: Crash Free Sessions / Users dropdown](release_crash_free.png)
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:51]: same comment as above re the screenshot

File: src/docs/product/releases/health/index.mdx [hunk 5]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
 
 Sessions are submitted to Sentry so you can track the usage and adoption of your application. When a user of your application experiences a crash, error, or abnormal exit, the session will be flagged accordingly, and Sentry calculates derived metrics. The metrics include data such as the number of users that didn't experience a crash in the specified time range.
 
-### Active Users
+### Active Sessions / Users
 
-Users who started the application at least once in the last 24 hours.
+Active Sessions are sessions that started in the application in the last 24 hours.
+
+Active Users are users who started the application at least once in the last 24 hours.
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:33]:
```suggestion
Active sessions are sessions that started in the application in the last 24 hours.

Active users are users who started the application at least once in the last 24 hours.
```

File: src/docs/product/releases/health/index.mdx [hunk 6]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
...
+To sort releases by active sessions, select Display: Sessions and to sort by active users select Display: Users.
+
+![Sort By: Active Sessions / Users dropdown](release_active.png)
 
 ### Crash
 
 The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `error.unhandled:true `. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) or [Sampling](/platform-redirect/?next=/configuration/sampling/).
 
-### Crash Free User
+### Crash Free Sessions / Users
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:43]:
```suggestion
### Crash Free Sessions/Users
```

File: src/docs/product/releases/health/index.mdx [hunk 7]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
...
+
+Crash Free Sessions is the percentage of sessions in the specified time range not ended by a crash of the application.
 
-Percentage of distinct users who did not experience a crash during the specified time period.
+Crash Free Users is the percentage of distinct users who did not experience a crash during the specified time period.
 
-### Crash Free Sessions
+To sort releases by crash free sessions, select Display: Sessions and to sort by crash free users select Display: Users.
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:49]:
```suggestion
Depending on your your "Display" selection, the sort options for the page change. To sort releases by crash free sessions, select "Sessions" in the "Display" dropdown, and to sort by crash free users, select "Users".
```

File: src/docs/product/releases/health/index.mdx [hunk 8]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
...
+![Sort By: Active Sessions / Users dropdown](release_active.png)
 
 ### Crash
 
 The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `error.unhandled:true `. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) or [Sampling](/platform-redirect/?next=/configuration/sampling/).
 
-### Crash Free User
+### Crash Free Sessions / Users
+
+Crash Free Sessions is the percentage of sessions in the specified time range not ended by a crash of the application.
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:45]: To get around the issue of "crash free sessions" being plural followed by a singular verb, I've put quotations around it and kept it capitalized to denote that we're referring to the (singular) UI element. 

```suggestion
"Crash Free Sessions" is the percentage of sessions in the specified time range not ended by a crash of the application.
```

File: src/docs/product/releases/health/index.mdx [hunk 9]
```diff
@@ -26,21 +26,29 @@ A session ends with the closing of the application or with the application being
...
 
-Percentage of distinct users who did not experience a crash during the specified time period.
+Crash Free Users is the percentage of distinct users who did not experience a crash during the specified time period.
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:47]:
```suggestion
"Crash Free Users" is the percentage of distinct users who did not experience a crash during the specified time period.
```

**Review by imatwawana — APPROVED** [review]
Really solid! Biggest suggestions are some changes to the screenshots and a couple sentences I added. Otherwise just some style and wording edits.

File: src/docs/product/releases/health/index.mdx (hunk 1 above)
Comment (taylangocmen) [reply src/docs/product/releases/health/index.mdx:37]: 👍

**Review by imatwawana — APPROVED** [review]
Looks good!

**Pull request merged**